packages/components/CHANGELOG.md

```diff
@@ -33,6 +34,16 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 + <p-button-pure>Some label</p-button-pure>
 ```
 
+- Rename prop `state`'s value `neutral` to `info` for `Inline Notification`, `Banner`, `Toast`
+
+```diff
+- <p-inline-notification heading="Some heading" description="Some description" state="neutral"></p-inline-notification>
++ <p-inline-notification heading="Some heading" description="Some description" state="info"></p-inline-notification>
+
+- <p-inline-notification state="neutral"></p-inline-notification>
++ <p-inline-notification state="info"></p-inline-notification>
+```
```

Comment:
Banner diffs might be good to have.  
Framework diffs might be helpful for toast, too.